lxc/export.go

```diff
@@ -107,8 +108,8 @@ func (c *cmdExport) Run(cmd *cobra.Command, args []string) error {
 	}
 
 	// Get name of backup
-	backupName := strings.TrimPrefix(op.Get().Resources["backups"][0],
-		"/1.0/backups/")
+	backupNameWithQueryParams := strings.TrimPrefix(op.Get().Resources["backups"][0], api.NewURL().Path(version.APIVersion, "instances", name, "backups").String()+"/")
```

Comment:
I'm really not a fan of this type of manual URL manipulation, as its error prone and does not take into account path escaping.

Can you do something like this: Parse the URL, extract escaped path, get last path segment, unescape it.

https://go.dev/play/p/8-W5iaNdsxO